src/persist-client/src/batch.rs

```diff
@@ -787,6 +821,11 @@ where
                 if (min_part_k, min_part_v, &min_part_t) < (max_run_k, max_run_v, max_run_t) {
                     soft_panic_or_log!("expected data in sorted order");
                     self.runs.push(self.parts_written);
+                    self.run_meta.push(
+                        self.parts
+                            .cfg
+                            .run_meta(RunOrder::Unordered, stats_schemas.id),
```

Comment:
Shouldn't this be Codec?

Reply:
This is the path we hit when the data was _supposed_ to be codec-ordered, but was unexpectedly out-of-order. So it seemed safest to declare the order with the fewest guarantees.

I don't think this panic has ever fired since it landed a few weeks ago, so I can probably turn it into a strict assert soon and avoid having to choose here either way...

Reply:
Ah, I just read the code wrong 👍